docs/contributing.md

```diff
@@ -1,9 +1,9 @@
 ## Contributions policy
 
-We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are taking issues and we do want your [feedback]({{urlRoot}}/index#give-us-feedback).
+We are currently not accepting public contributions (but we do plan to in later releases and we’ll keep you posted on this). However, we are accepting issues and we do want your [feedback](https://github.com/spatialos/gdk-for-unity#give-us-feedback).
```

Comment:
does `{{urlRoot}}/index` point to the welcome page or the README.md at root?

if it does, can revert the `index` fixes